python/ray/_private/worker.py

```diff
@@ -1091,6 +1092,7 @@ def init(
         namespace: Namespace to use
         runtime_env: The runtime environment to use
             for this job (see :ref:`runtime-environments` for details).
+        metadata (list): List of tuples representing request metadata for Ray Client gRPC.
```

Comment:
Looks like the [lint](https://buildkite.com/ray-project/ray-builders-pr/builds/36780#0181a754-2fb9-4014-8df3-3903139bfac3) is failing on here, can you remove the `(list)` part

Reply:
@ckw017  Sure thing! Resolved. Sorry, not used to CI in Github.